common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -5302,7 +5302,7 @@ object CelebornConf extends Logging {
     buildConf("celeborn.quota.tenant.diskBytesWritten")
       .categories("quota")
       .dynamic
-      .doc("Quota dynamic configuration for written disk bytes.")
+      .doc("Quota dynamic configuration for written disk bytes. 0 means that no shuffle submissions are allowed.")
```

Comment:
You can add value check for this configuration, to make sure value in that range, like

`   .checkValue(v => v > =0, "Should be non-negative numbers.")
`